R/bootstrap.R

```diff
@@ -44,6 +47,16 @@ bootstrapPage <- function(..., title = NULL, responsive = NULL, theme = NULL) {
     ),
     bootstrapLib()
   )
+
+  # Also store `lang` to be passed for rendering processor.
+  if (!is.null(lang)) {
```

Comment:
We can get rid of all these `if` statements and assign the attribute unconditionally